src/WellIO.jl

```diff
@@ -1,5 +1,47 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+"""
+    load(file)
+Load log well file from `filepath` and load it in defined structure.
+"""
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+            read(f, String)
+        end
+    #Splitting the data into table and rest of information
+    datatable_block = match(r"~A(.*?)\Z"s, data)
+    info_block = data[1:datatable_block.offset]
```

Comment:
Use a blank line to separate different sections of the code. Your code formatting conventions need improvement.